src/Channel.php

```diff
@@ -63,7 +64,7 @@ public function __construct(
         $this->supervisor = new DeliverySupervisor($this, $this->hooks, $this->channelId);
         $this->consumerTags = new ConsumerTagGenerator();
         $this->consumer = Consumer::create($this->supervisor, $this);
-        $this->receiver = Receiver::create($this->supervisor);
+        $this->gets = new ChannelAtomicGet(Receiver::create($this->supervisor), $this->connection, $this->channelId);
```

Comment:
```suggestion
        $this->get = new ChannelAtomicGet(Receiver::create($this->supervisor), $this->connection, $this->channelId);
```

There's only one get "in fly" there?